Localization/Resources/es-ES/winget.resw

```diff
@@ -1268,4 +1275,78 @@ Especifique uno de ellos con la opción '--source' para continuar.</value>
   <data name="SystemArchitecture" xml:space="preserve">
     <value>Arquitectura del sistema</value>
   </data>
+  <data name="PreserveArgumentDescription" xml:space="preserve">
+    <value>Conserva todos los archivos y directorios creados por el paquete (archivo portátil)</value>
+  </data>
+  <data name="PurgeArgumentDescription" xml:space="preserve">
+    <value>Elimina todos los archivos y directorios del directorio del paquete (archivo portátil)</value>
+  </data>
+  <data name="RenameArgumentDescription" xml:space="preserve">
+    <value>Valor para cambiar el nombre del archivo ejecutable (archivo portátil)</value>
+  </data>
+  <data name="WaitArgumentDescription" xml:space="preserve">
+    <value>Pide al usuario que presione cualquier tecla antes de salir</value>
+  </data>
+  <data name="InstallerElevationExpected" xml:space="preserve">
+    <value>El instalador solicitará que se ejecute como administrador; se espera un mensaje.</value>
```

Comment:
Instead of "se espera un mensaje", I think it should be "espere un mensaje". I.e., instead of "a prompt is expected", something more along the lines of "you should expect a prompt". I'm also not sure if "prompt" should be "mensaje" in this case? Maybe "cuadro de diálogo"?